### PR TITLE
Added extra use cases and unused files

### PR DESCRIPTION
**Pull request by daongochieu2810**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -961,6 +961,16 @@ For all use cases below, the **System** is the `Wishful Shrinking` and the **Act
   1. Wishful Shrinking shows all valid commands.
 
      Use case ends.
+**Use case: Select a recipe in the recipe list**
+
+**MSS**
+
+  1. User requests to view a single recipe.
```
Comment (caitlinjee) [inline docs/DeveloperGuide.md:1112]: i think can add extension when index is not correct?

**Review by caitlinjee — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (TianYong-Goh) [reply docs/DeveloperGuide.md:1112]: Agree.

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Review by oliviajohansen — APPROVED** [review]
lgtm

**Review by jiaqi20 — APPROVED** [review]
lgtm

**Pull request merged**